src/DDTrace/Configuration/EnvVariableRegistry.php

```diff
@@ -18,21 +18,52 @@ public function __construct()
         $this->registry = [];
     }
 
+    /**
+     * Return an env variable that starts with "DD_".
+     *
+     * @param string $key
+     * @return string|null
+     */
+    public static function get($key)
+    {
+        $value = getenv(self::convertKeyToEnvVariableName($key));
+        if (false === $value) {
+            return null;
+        }
+        return trim($value);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function stringValue($key, $default)
```

Comment:
Should we honor `$default`? :)

Reply:
Ha! Good catch! I totally missed that one! :D